### ISSUE TITLE
plotScoreHeatmap color parameter

### ISSUE THREAD

**Issue by alanaleah89** [LTLA/SingleR#197]
Hi there,

I would like to use my own colour scheme for SingleRs heatmap. I think there may be a bug in the plotScoreHeatmap method.
When I execute the following code I get the same color scheme as the default.
plotScoreHeatmap(p.abx, color = (grDevices::colorRampPalette(c('#f1a340','#f7f7f7','#998ec3')))(100))
![p abx](https://user-images.githubusercontent.com/26641213/125710751-8bfdfd04-3ebc-4073-9bb3-a8591ca65516.png)
However, if I execute the same code with normalize = F, I get the color scheme I want but I would also like my data normalised. 
plotScoreHeatmap(p.abx, color = (grDevices::colorRampPalette(c('#f1a340','#f7f7f7','#998ec3')))(100), normalize = F)
![p abx](https://user-images.githubusercontent.com/26641213/125710944-16ce2b4a-885e-4fef-9436-eb71b6e775d3.png)

**Comment by LTLA** [comment]
@dtm2451 can you have a look at this?

**Comment by dtm2451** [comment]
Yes, yes.  Sorry for the delay.. back from a long vacation today!

I know where this is from and can definitely address it this week **if we want to** @LTLA.  My only concern is that it doesn't necessarily make sense for a user to use a bipolar color scale for the normalized data.  I can't remember if we had kept the colors out of a users hands *on purpose* when `normalize = TRUE`.  But it would feel like a bug to users anyway.  I'll make a PR for this (and any other pheatmap inputs that may be overridden internally, if there are any).

**Comment by LTLA** [comment]
Right... yeah, I guess we should at least respond to the user-supplied `color` when `normalize=TRUE`, and it's on the user to make sure they choose a sensible color scale. In this case, a bidirectional one probably doesn't make a lot of sense, but I suppose someone might prefer something other than viridis (_gasp_), in which case we should support that.

**Comment by lindaboshans** [comment]
Following up on this. I also can not use custom colors when normalize is set to T.

**Comment by LTLA** [comment]
@dtm2451 did we decide what to do here?

**Comment by dtm2451** [comment]
Thanks for the bump @lindaboshans.
@LTLA looks to me above like we decided to give users this control, and leave it to them to pick a color scale that makes sense. ...I just forgot to do so.  Starting this work now!

**Comment by LTLA** [comment]
Pushed to 2.4.1, should be available in a few days.
